command/server/config_test_helpers.go

```diff
@@ -711,6 +711,12 @@ listener "tcp" {
 	tls_max_version = "tls13"
 	tls_require_and_verify_client_cert = true
 	tls_disable_client_certs = true
+    telemetry {
+      unauthenticated_metrics_access = true
```

Comment:
There are a bunch of config fields where we are using `enable_` and `disable_` as prefixes, including the newly added `enable_response_header_raft_node_id` and the `enable_response_header_hostname`.

These ones also seem to fall under the same category and could take `enable_` prefixes. WDYT?

Reply:
Also, just to be sure that this is not something are overlooking. Is there a reason why these config shouldn't live at the top level (as opposed to being in the listener blocks)?

Reply:
Re enable/disable: a boolean type already implies enabling/disabling something to me, and we have many bools already that don't have such prefixes.  In any event, in this case the new setting mirrors unauthenticated_metrics_access and I'd rather be consistent with that.

These configs belong in listener blocks rather than top-level so that people can enable them in e.g. localhost listener but not in public listener, as a safety precaution.

Reply:
Sounds good!